roboslack-api/src/main/java/com/palantir/roboslack/api/markdown/MrkdwnIn.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.roboslack.api.markdown;
+
+import java.util.Arrays;
+import java.util.Optional;
+import javax.annotation.CheckForNull;
+
+/**
+ * Enumerated types for handling the mrkdwn_in flag in Slack message formatting.
+ *
+ * @since v0.2.2
+ */
+public enum MrkdwnIn {
+    /**
+     * Pretext.
+     */
+    PRETEXT("pretext"),
+    /**
+     * Text.
+     */
+    TEXT("text"),
+    /**
+     * Fields.
+     */
+    FIELDS("fields");
```

Comment:
The `String value` property is probably not needed, just need to `toLowerCase()` the `value()`